### DISCUSSION TITLE
Estimated Allowed Amount Clarification

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#36] [Q&A] [answered]
Hi All. Needing help clarifying when the Estimated Allowed Amount is required to be in the MRF. Per the Hospital Machine-Readable File Data Dictionary page, under the Conditional Requirements section, #6 says "If a "payer specific negotiated charge" can only be expressed as a percentage or algorithm, then a corresponding "Estimated Allowed Amount" must also be encoded. Required beginning 1/1/2025."

Does this mean the ONLY TIME the MRF needs the Estimated Allowed Amount is when the negotiated charge is expressed as a percentage or algorithm? AND any other time it does NOT need to be included? 

Otherwise why would it be called out in #6, if it was required all the time.

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question. 

As indicated in the CY 2024 OPPS/ASC Final Rule, unless otherwise specified, beginning July 1, 2024, each hospital must encode in its machine-readable file all standard charge information, as applicable, for the required data elements, including whether the standard charge indicated should be interpreted by the user as a dollar amount, or if the standard charge is based on a percentage or algorithm. If the standard charge is based on a percentage or algorithm, the machine-readable file (MRF) must also describe the percentage or algorithm that determines the dollar amount for the item or service, and, beginning January 1, 2025, calculate and encode an estimated allowed amount in dollars for that item or service. Please refer to [45 CFR § 180.50 (b)(2)(ii)(C).](https://www.ecfr.gov/current/title-45/part-180/section-180.50#p-180.50(b)(2)(ii)(C))
